server/application-server/src/main/java/de/tum/in/www1/hephaestus/Application.java

```diff
@@ -1,15 +1,31 @@
 package de.tum.in.www1.hephaestus;
 
+import java.util.List;
+
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.modulith.Modulithic;
 
+import de.tum.in.www1.hephaestus.codereview.CodeReviewService;
+import de.tum.in.www1.hephaestus.codereview.repository.Repository;
+
 @SpringBootApplication
 @Modulithic(systemName = "Hephaestus")
+@EnableConfigurationProperties(EnvConfig.class)
 public class Application {
 
 	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
 	}
 
+	@Bean
+	CommandLineRunner commandLineRunner(CodeReviewService service) {
+		return args -> {
+			Repository repo = service.fetchRepository("ls1intum/hephaestus");
```

Comment:
Fetching on startup makes sense to me but maybe this should also be configurable using `@Value`, same with the repositories to fetch.

Reply:
Example how you could move it out of the `CommandLineRunner` and use `Value`:
```
@Value("${monitoring.runOnStartup:true}")
private boolean runOnStartup;

@PostConstruct
public void onStartup() {
    if (runOnStartup) {
        doWork();
    }
}
```

I think this would be preferred